src/FakeItEasy.Extensions.ValueTask/FakeItEasy.Extensions.ValueTask.csproj

```diff
@@ -42,26 +42,23 @@
     <ProjectReference Include="..\FakeItEasy\FakeItEasy.csproj" />
   </ItemGroup>
 
-  <ItemGroup Condition="!$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'netstandard2.1'))">
+  <ItemGroup Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net462'))">
     <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.3" />
   </ItemGroup>
 
   <!-- Selectively include build output for package -->
   <ItemGroup>
     <None Include="$(OutputPath)/**/$(AssemblyName).dll" Pack="true" PackagePath="lib" Visible="false" />
     <None Include="$(OutputPath)/**/$(AssemblyName).xml" Pack="true" PackagePath="lib" Visible="false" />
-    <None Remove="$(OutputPath)/netstandard2.1/**" />
     <None Remove="$(OutputPath)/net6.0/**" />
     <None Remove="$(OutputPath)/net8.0/**" />
   </ItemGroup>
 
-  <!-- Include placeholder files in netstandard2.1 and netX.Y folders in package -->
+  <!-- Include placeholder files in netX.Y folders in package -->
   <Target Name="AddTfmPlaceholders">
-    <WriteLinesToFile File="$(IntermediateOutputPath)netstandard2.1/_._" Overwrite="true" Lines="" />
     <WriteLinesToFile File="$(IntermediateOutputPath)net6.0/_._" Overwrite="true" Lines="" />
     <WriteLinesToFile File="$(IntermediateOutputPath)net8.0/_._" Overwrite="true" Lines="" />
     <ItemGroup>
-      <None Include="$(IntermediateOutputPath)netstandard2.1/_._" Pack="true" PackagePath="lib/netstandard2.1" Visible="false" />
       <None Include="$(IntermediateOutputPath)net6.0/_._" Pack="true" PackagePath="lib/net6.0" Visible="false" />
       <None Include="$(IntermediateOutputPath)net8.0/_._" Pack="true" PackagePath="lib/net8.0" Visible="false" />
     </ItemGroup>

```

Comment:
I was looking at this yesterday and wondering if we could avoid hard-coding the TFMs for which we need a placeholder; it's a bit strange that we use a MSBuild property for `TargetFrameworks`, but hard-code the TFMs here. But maybe it's not worth the trouble, since it will soon be just net8.0...